src/code/templates/fv-egg-game.js

```diff
@@ -93,12 +93,21 @@ function lookupGameteChromosomeDOMElement(sex, chromosomeName) {
   return genomeWrapper.querySelectorAll(".fv-chromosome-image")[chromosomePositions[chromosomeName]];
 }
 
-function findBothElements(sex, name, el){
+function unscaleProperties(obj, scale=1) {
```

Comment:
Probably, but we can figure out in what form when we need to move beyond this first challenge. It may be that it can get packaged in some coherent animation wrapper component.